pyk/src/pyk/cli/args.py

```diff
@@ -475,6 +477,9 @@ def kompile_args(self) -> ArgumentParser:
             action='store_true',
             help='Do not wrap the output on the CLI.',
         )
+        args.add_argument(
+            '--ignore-warnings', dest='ignore_warnings', default=None, action='append', help='Ignore provided warnings'
```

Comment:
If we want to be compatible with the `kompile` CLI, we need to support the way `kompile` consumes this option.

@JuanCoRo, can you check which of the above methods `kompile` supports to take multiple warnings to ignore?

Reply:
This probably calls for a custom action that instead of `append`ing to the list `extend`s it. I'm not sure how well our current CLI convention supports this though.

Anyways, this doesn't have to be addressed in this PR. @JuanCoRo, could you open an issue for tracking this?

Reply:
I don't follow what the problematic behavior here is. Could you elaborate a bit for me to see it @tothtamas28? Thanks!

Reply:
```
-Wno non-exhaustive-match -Wno missing-syntax-module,deprecated-symbol
```

will parse to

```python3
['non-exhaustive-match', 'missing-syntax-module,deprecated-symbol']
```

which is fine in the sense that concatenating them with `,` will give you something you can pass to `kompile`. Would still be nice to parse the atoms separately:

```python
['non-exhaustive-match', 'missing-syntax-module', 'deprecated-symbol']
```

Reply:
Just opened #4479